Clarify and correct initialization content re: Docker (#2229)
sensu/sensu-docs: content/sensu-go/5.18/reference/rbac.md

### Default users

> During the [Sensu backend installation][42] process, you can set administrator credentials to override the default credentials for a `default` namespace.

Reviewer: ```suggestion
During the [Sensu backend installation][42] process, you can set administrator credentials to override the default credentials and a `default` namespace.
```

### Default users

> During the [Sensu backend installation][42] process, you can set administrator credentials to override the default credentials and create a `default` namespace.

Reviewer: ```suggestion
During the [Sensu backend installation][42] process, you create an administrator username and password and a `default` namespace.
```
